src/component/Todo/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from '@/libs/myReact'
+import './todo.css'
+
+export default function Index() {
+	const [input, setInput] = useState<string>('')
+	const [todos, setTodos] = useState<string[]>([])
+
+	const handleChange = (e: any) => {
+		setInput(e.target.value)
+	}
+	const handleClick = () => {
+		setTodos([...todos, input])
+		setInput('')
+	}
+
+	const handleDelete = (index: number) => {
+		setTodos(todos.filter((_, i) => i !== index))
+	}
+
+	useEffect(() => {
+		console.log('useEffect - todos:', todos)
+		return () => {
+			console.log('useEffect cleanup - todos:', todos)
+		}
+	}, [todos])
+
+	console.log('??')
+
+	return (
+		<div>
+			<div>현재 카운트는 {todos.length} 입니다.</div>
+			<TodoForm
+				input={input}
+				handleChange={handleChange}
+				handleClick={handleClick}
+			/>
+			<TodoList todos={todos} handleDelete={handleDelete} />
+		</div>
+	)
+}
+
+interface TodoFormProps {
+	input: string
+	handleChange: (e: any) => void
+	handleClick: () => void
```

Comment:
컴포넌트가 이벤트 핸들러를 props로 받을 때는 on- prefix로 표현해주시는 게 좋을 것 같아요~
```suggestion
	value: string
	onChange: (e: any) => void
	onClick: () => void
```